src/hooks/safe-apps/useSafeAppFromManifest.ts

```diff
@@ -9,17 +9,37 @@ type UseSafeAppFromManifestReturnType = {
   isLoading: boolean
 }
 
+const UNKNOWN_APP_NAME = 'unknown'
+const UNKNOWN_APP_LOGO = '/images/apps-icon.svg'
+
 const useSafeAppFromManifest = (appUrl: string, chainId: string): UseSafeAppFromManifestReturnType => {
   const [data, error, loading] = useAsync<SafeAppData>(() => {
     if (appUrl && chainId) return fetchSafeAppFromManifest(appUrl, chainId)
   }, [appUrl, chainId])
 
+  const emptyApp = useMemo(() => getEmptySafeApp(appUrl), [appUrl])
+
   useEffect(() => {
     if (!error) return
     logError(Errors._900, `${appUrl}, ${(error as Error).message}`)
   }, [appUrl, error])
 
-  return { safeApp: data, isLoading: loading }
+  return { safeApp: data || emptyApp, isLoading: loading }
```

Comment:
Why is an empty app needed?

Reply:
That's for providing a barebone for starting the communicator as sometimes the manifests are blocked by CORS or simply the request can fail

https://github.com/safe-global/web-core/blob/07931c92964687d18f70963f05b8a12bef40ffa9/src/components/safe-apps/AppFrame/useAppCommunicator.ts#L58

But now that you mention this perhaps I can remove completely the app as mandatory (Already removed from AppCommunicator)

Reply:
Done, seems to work nicely. If I detect some problem when testing the next PR (sign and transactions) I will revert this

Reply:
I think I've added it to solve some nasty bug in some rare case where I had to reinitialize the communicator

Reply:
>That's for providing a barebone for starting the communicator as sometimes the manifests are blocked by CORS or simply the request can fail

If the manifest is blocked or the request is failed, we don't want to start the communicator IMO, it feels very hacky to me

Reply:
> I think I've added it to solve some nasty bug in some rare case where I had to reinitialize the communicator

Ah no, it seems it's only needed for error logging, nevermind..

Reply:
Ok, let's avoid the initialisation if we don't have an app then. Changed

Reply:
what a top senor you are